data/source/ulas_j033350.84+001406.1.json

```diff
@@ -16,6 +16,9 @@
         {
             "other_name": "2MASS J03335076+0014062"
         },
+        {
+            "other_name": "2MASS J03361338+0010129"
+        },
```

Comment:
maybe wrong match?  The numbers usually agree really well and these are just a bit more off than usual.